src/content/config.ts

```diff
@@ -43,16 +43,40 @@ export const collections = {
 	}),
 	blog: defineCollection({
 		schema: z.object({
-			title: z.string(),
-			description: z.string(),
+			title: z.string().describe('The blog post title.'),
+			description: z
+				.string()
+				.describe(
+					'Summary of this blog post. Appears on the blog index as well as in metadata displayed on social media.',
+				),
 			publishDate: z
 				.string()
 				.or(z.date())
-				.transform((val) => new Date(val)),
-			authors: z.array(z.string()),
-			socialImage: z.string().optional(),
-			coverImage: z.string().optional(),
-			lang: z.enum(['en']).default('en'),
+				.transform((val) => new Date(val))
```

Comment:
It is, yes. I considered changing it, but we’d get misleading errors in the intellisense currently (because we generally don’t use a YAML-compatible date for these). I can still change it though if you think it’s OK? In any case it’s always consumed as a JS `Date`, so authoring style is kind of irrelevant.

Reply:
OK, refactored to use `z.coerce.date()`. Up to people if they ignore the warnings or not when authoring.